edgelet/edgelet-core/src/error.rs

```diff
@@ -73,13 +73,13 @@ pub enum ErrorKind {
     InvalidModuleType(String),
 
     #[fail(
-        display = "Error parsing URI {} specified for '{}'. Please check the config.yaml file.",
+        display = "Error parsing URI {} specified for '{}'. Please check the config.toml file.",
```

Comment:
"config.toml" isn't right either; there are multiple files. Make it "configuration" like what `aziotctl check` calls it.

Actually some of these are probably unused now, so just delete them them.

Reply:
fixed